v2/bundle/manifests/redhat-marketplace-operator.clusterserviceversion.yaml

```diff
@@ -1172,7 +1172,8 @@ spec:
       targetPort: 9443
       type: MutatingAdmissionWebhook
       webhookPath: /mutate-marketplace-redhat-com-v1beta1-meterdefinition
-    - admissionReviewVersions: null
+    - admissionReviewVersions: 
```

Comment:
This code is generated - I'm pretty sure this is going to be overwritten. This is probably one the webhook annotations found in this file v2/apis/marketplace/v1beta1/meterdefinition_webhook.go